src/components/DefineCompany/SummaryDefineCompany.vue

```diff
@@ -28,6 +28,14 @@
             <span v-else-if="entityFilter(EntityTypes.COOP)">BC Cooperative Association</span>
           </div>
         </v-flex>
+<!--        *** THIS IS AN EXAMPLE OF IMPLEMENTATION FOR DEVELOPMENT ONLY-->
```

Comment:
Sorry for the change in my other PR. This code change will now live in `YourCompany.vue`.

Reply:
Not a problem! Do you think it's ok if i just remove this comment now? I'm fairly certain implementation isn't too complicated, this was more for some context.

Reply:
Absolutely. And feel free to remove "TODO" comments in my code as things get done/integrated. 👍